CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## Coming Next
+## 0.4.0
+
+**changelog won't be updated since 0.4.0, please check the release page:**
```

Comment:
I found the github generated release notes are very good.

However, is there an easy way to commit it into the repo? in this changelog.txt so that we can include it to the source code distribution. @messense

Reply:
I think it's doable, just make a action which triggers after each release, then use the GitHub API to read the release note and commit it into the repo.

For example the release note is in the `body` field:

```bash
root@raspberrypi:~# curl \
>   -H "Accept: application/vnd.github.v3+json" \
>   https://api.github.com/repos/PyO3/maturin/releases/latest
{
  "url": "https://api.github.com/repos/PyO3/maturin/releases/54708076",
  "assets_url": "https://api.github.com/repos/PyO3/maturin/releases/54708076/assets",
  "upload_url": "https://uploads.github.com/repos/PyO3/maturin/releases/54708076/assets{?name,label}",
  "html_url": "https://github.com/PyO3/maturin/releases/tag/v0.12.4",
  "id": 54708076,
  "author": {
    "login": "messense",
    "id": 1556054,
    "node_id": "MDQ6VXNlcjE1NTYwNTQ=",
    "avatar_url": "https://avatars.githubusercontent.com/u/1556054?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/messense",
    "html_url": "https://github.com/messense",
    "followers_url": "https://api.github.com/users/messense/followers",
    "following_url": "https://api.github.com/users/messense/following{/other_user}",
    "gists_url": "https://api.github.com/users/messense/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/messense/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/messense/subscriptions",
    "organizations_url": "https://api.github.com/users/messense/orgs",
    "repos_url": "https://api.github.com/users/messense/repos",
    "events_url": "https://api.github.com/users/messense/events{/privacy}",
    "received_events_url": "https://api.github.com/users/messense/received_events",
    "type": "User",
    "site_admin": false
  },
  "node_id": "RE_kwDOCHRpys4DQsds",
  "tag_name": "v0.12.4",
  "target_commitish": "main",
  "name": "v0.12.4",
  "draft": false,
  "prerelease": false,
  "created_at": "2021-12-06T13:05:31Z",
  "published_at": "2021-12-06T13:11:37Z",
  "assets": [
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125496",
      "id": 51125496,
      "node_id": "RA_kwDOCHRpys4DDBz4",
      "name": "maturin-aarch64-apple-darwin.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 5924313,
      "download_count": 2,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:11Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-aarch64-apple-darwin.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125506",
      "id": 51125506,
      "node_id": "RA_kwDOCHRpys4DDB0C",
      "name": "maturin-aarch64-pc-windows-msvc.zip",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/zip",
      "state": "uploaded",
      "size": 4676947,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:11Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-aarch64-pc-windows-msvc.zip"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125498",
      "id": 51125498,
      "node_id": "RA_kwDOCHRpys4DDBz6",
      "name": "maturin-aarch64-unknown-linux-musl.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 8630842,
      "download_count": 4,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:11Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-aarch64-unknown-linux-musl.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125499",
      "id": 51125499,
      "node_id": "RA_kwDOCHRpys4DDBz7",
      "name": "maturin-armv7-unknown-linux-musleabihf.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 8976232,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-armv7-unknown-linux-musleabihf.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125507",
      "id": 51125507,
      "node_id": "RA_kwDOCHRpys4DDB0D",
      "name": "maturin-i686-pc-windows-msvc.zip",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/zip",
      "state": "uploaded",
      "size": 5037395,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-i686-pc-windows-msvc.zip"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125497",
      "id": 51125497,
      "node_id": "RA_kwDOCHRpys4DDBz5",
      "name": "maturin-i686-unknown-linux-musl.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 9345327,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-i686-unknown-linux-musl.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125503",
      "id": 51125503,
      "node_id": "RA_kwDOCHRpys4DDBz_",
      "name": "maturin-musl_0.12.4_amd64.deb",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/octet-stream",
      "state": "uploaded",
      "size": 4438144,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-musl_0.12.4_amd64.deb"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125501",
      "id": 51125501,
      "node_id": "RA_kwDOCHRpys4DDBz9",
      "name": "maturin-powerpc64le-unknown-linux-musl.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 7837928,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-powerpc64le-unknown-linux-musl.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125500",
      "id": 51125500,
      "node_id": "RA_kwDOCHRpys4DDBz8",
      "name": "maturin-s390x-unknown-linux-gnu.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 9211001,
      "download_count": 0,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-s390x-unknown-linux-gnu.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125504",
      "id": 51125504,
      "node_id": "RA_kwDOCHRpys4DDB0A",
      "name": "maturin-x86_64-apple-darwin.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 6384159,
      "download_count": 165,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-x86_64-apple-darwin.tar.gz"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125505",
      "id": 51125505,
      "node_id": "RA_kwDOCHRpys4DDB0B",
      "name": "maturin-x86_64-pc-windows-msvc.zip",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/zip",
      "state": "uploaded",
      "size": 5494722,
      "download_count": 238,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-x86_64-pc-windows-msvc.zip"
    },
    {
      "url": "https://api.github.com/repos/PyO3/maturin/releases/assets/51125502",
      "id": 51125502,
      "node_id": "RA_kwDOCHRpys4DDBz-",
      "name": "maturin-x86_64-unknown-linux-musl.tar.gz",
      "label": "",
      "uploader": {
        "login": "github-actions[bot]",
        "id": 41898282,
        "node_id": "MDM6Qm90NDE4OTgyODI=",
        "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/github-actions%5Bbot%5D",
        "html_url": "https://github.com/apps/github-actions",
        "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
        "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
        "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
        "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
        "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
        "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
        "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
        "type": "Bot",
        "site_admin": false
      },
      "content_type": "application/gzip",
      "state": "uploaded",
      "size": 9079845,
      "download_count": 540,
      "created_at": "2021-12-06T13:26:11Z",
      "updated_at": "2021-12-06T13:26:12Z",
      "browser_download_url": "https://github.com/PyO3/maturin/releases/download/v0.12.4/maturin-x86_64-unknown-linux-musl.tar.gz"
    }
  ],
  "tarball_url": "https://api.github.com/repos/PyO3/maturin/tarball/v0.12.4",
  "zipball_url": "https://api.github.com/repos/PyO3/maturin/zipball/v0.12.4",
  "body": "## What's Changed\r\n* Bump anyhow from 1.0.50 to 1.0.51 by @dependabot in https://github.com/PyO3/maturin/pull/717\r\n* init: new command similar to `cargo init` by @davidhewitt in https://github.com/PyO3/maturin/pull/719\r\n* Don't package non-path-dep crates in sdist for workspaces by @messense in https://github.com/PyO3/maturin/pull/720\r\n* Update keyring to 1.0.0 by @messense in https://github.com/PyO3/maturin/pull/721\r\n* Build release packages with password-storage feature by @messense in https://github.com/PyO3/maturin/pull/725\r\n* Reorganize upload feature by @messense in https://github.com/PyO3/maturin/pull/726\r\n* Add support for x86_64 DragonFly BSD by @messense in https://github.com/PyO3/maturin/pull/727\r\n* Limit concurrency on GitHub Actions by @messense in https://github.com/PyO3/maturin/pull/728\r\n* Allow pip warnings in maturin develop command by @messense in https://github.com/PyO3/maturin/pull/732\r\n* Add a Python import hook by @messense in https://github.com/PyO3/maturin/pull/729\r\n\r\n\r\n**Full Changelog**: https://github.com/PyO3/maturin/compare/v0.12.3...v0.12.4",
  "discussion_url": "https://github.com/PyO3/maturin/discussions/733",
  "mentions_count": 3
}
```

Reply:
OK so maybe I can create a new github action to do this.

Reply:
@messense I realized another problem: by such way, we can only commit changelogs after a git tag is created.

So in a new tagged version, the current version's changelog won't be included. TT